### PR TITLE
add the missing option to let user exclude dot files

### PR DESCRIPTION
**Pull request by maboiteaspam**
see #31

### CODE REVIEW

File: tasks/sftp-deploy.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ module.exports = function(grunt) {
     files = fs.readdirSync(startDir);
     for (i = 0; i < files.length; i++) {
       currFile = startDir + path.sep + files[i];
-      if (!file.isMatch({matchBase: true}, exclusions, currFile)) {
+      if (!file.isMatch({matchBase: true,dot:true}, exclusions, currFile)) {
```
Comment (thrashr888) [inline tasks/sftp-deploy.js]: just a nitpick that there should be a space after the comma.

**Comment by thrashr888** [comment]
I think that some users might want to transfer their hidden files. This is probably better as an option rather than forcing it for all uploads.

**Comment by maboiteaspam** [comment]
So old. I totally forgot about this. Indeed, i agree an option would be better. For syntax i suggest eslint for code style check.

**Comment by thrashr888** [comment]
good idea to add a jshint config file. thanks.

**Comment by maboiteaspam** [comment]
eslint much better in my mind : ) Have an example here https://github.com/cozy-labs/cozy-light/blob/master/.eslintrc and have the website there http://eslint.org/docs/rules/

**Pull request closed without merging**